onnx/backend/test/case/node/split.py

```diff
@@ -113,7 +117,165 @@ def export_default_values() -> None:
             node,
             inputs=[input],
             outputs=[y for y in expected_outputs],
-            name="test_split_equal_parts_default_axis",
+            name="test_split_equal_parts_default_axis_opset13",
+            opset_imports=[onnx.helper.make_opsetid("", 13)],
+        )
+
+        split = np.array([2, 4]).astype(np.int64)
+        node = onnx.helper.make_node(
+            "Split", inputs=["input", "split"], outputs=["output_1", "output_2"]
+        )
+
+        expected_outputs = [
+            np.array([1.0, 2.0]).astype(np.float32),
+            np.array([3.0, 4.0, 5.0, 6.0]).astype(np.float32),
+        ]
+        expect(
+            node,
+            inputs=[input, split],
+            outputs=[y for y in expected_outputs],
+            name="test_split_variable_parts_default_axis_opset13",
+            opset_imports=[onnx.helper.make_opsetid("", 13)],
+        )
+
+    @staticmethod
+    def export_zero_size_splits_opset13() -> None:
+        input = np.array([]).astype(np.float32)
```

Comment:
Can you clarify the input/output shapes explicitly to help the reader? I assume that the shapes are [0] (a 1-dimensional tensor with dimension-size=0).